### PR TITLE
feat(writer): update commit types

### PR DESCRIPTION
**Pull request by tvardy**
Just an update of commit types / categories based on [conventional-changelog types](https://github.com/commitizen/conventional-commit-types/blob/master/index.json)

### CODE REVIEW

**Comment by dmlittle** [comment]
Hey @tvardy, this looks good! Could you also add these new categories to the [README.md](https://github.com/lob/generate-changelog/blob/master/README.md#usage) in this PR?

**Comment by tvardy** [comment]
Will do ;-)

**Comment by tvardy** [comment]
README.md updated (I also added info about #15 / #17 as I've forgotten to od this previously)

File: lib/writer.js [hunk 1]
```diff
@@ -4,12 +4,16 @@ var Bluebird = require('bluebird');
 
 var DEFAULT_TYPE = 'other';
 var TYPES = {
+  build: 'Build system / dependencies',
```
Comment (robinjoseph08) [inline lib/writer.js]: To match the other types, can you capitalize all of the words? `Build System / Dependencies`

File: lib/writer.js [hunk 2]
```diff
@@ -4,12 +4,16 @@ var Bluebird = require('bluebird');
...
+  ci: 'Continuous Integration',
   chore: 'Chores',
   docs: 'Documentation Changes',
   feat: 'New Features',
   fix: 'Bug Fixes',
   other: 'Other Changes',
+  perf: 'Perfomance improvements',
```
Comment (robinjoseph08) [inline lib/writer.js]: Same as above, can you capitalize `Improvements`?

Also, looks like you misspelled `Performance` 😆

**Comment by tvardy** [comment]
fixed

File: lib/writer.js [hunk 3]
```diff
@@ -4,12 +4,16 @@ var Bluebird = require('bluebird');
 
 var DEFAULT_TYPE = 'other';
 var TYPES = {
+  build: 'Build system / Dependencies',
```
Comment (robinjoseph08) [inline lib/writer.js]: Looks like you missed `System` when you were capitalizing

Comment (tvardy) [inline lib/writer.js]: good spot 😂

**Comment by robinjoseph08** [comment]
Just published v1.3.0!

**Comment by tvardy** [comment]
👍

**Pull request merged**